src/pytest_rich/plugin.py

```diff
@@ -9,7 +10,32 @@
 
 
 def pytest_addoption(parser):
-    parser.addoption("--rich", action="store_true", default=False)
+    group = parser.getgroup("rich", "pytest-rich", after="terminal reporting")
+    group.addoption(
+        "--rich",
+        action="store_true",
+        default=False,
+        help="Enable rich terminal reporting using pytest-rich",
+    )
+    group.addoption(
+        "--rich-capture",
+        action="store_true",
+        default=False,
+        help="Capture terminal output using rich",
+    )
+    group.addoption(
+        "--rich-capture-file-name",
+        action="store",
+        default=f"{__name__.split('.')[0]}-{datetime.datetime.now().isoformat().replace(':', '.')}",
+        help="File name to use for rich terminal capture (only if --rich-capture is also enabled). Default is <module name>-<datetime>",
+    )
```

Comment:
Probably makes sense to turn these 3 flags into one? For example `--rich-capture=out.svg` would mean the same as `--rich-capture --rich-capture-file-name=out.svg --rich-capture-file-type=svg`?